views/article.html

```diff
@@ -49,20 +47,18 @@ <h2 class="article-standfirst">{{standfirst}}</h2>
 		</div>
 	</header>
 
-	<div amp-access="access OR {{showEverything}}" amp-access-hide>
-		<div class="article-body" itemprop="articleBody">
-			{{{htmlBody}}}
+	<div class="article-body" itemprop="articleBody">
```

Comment:
does this need a `subscriptions-display` or a `subscriptions-section` attribute? how does it get hidden?

Reply:
ditto for header & comments

Reply:
The `subscriptions-section="content"` bit here wraps around all of that and is what `amp-subscriptions` displays when access is granted: https://github.com/Financial-Times/google-amp/pull/279/files#diff-febb4d6e59192de3d32fc452273edc4fR162

Reply:
ah that makes sense

Reply:
ah that makes sense